cmd/ore/aws/upload.go

```diff
@@ -236,6 +238,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if len(uploadTags) > 0 {
+		hvmTagMap := make(map[string]string)
+		for _, tag := range uploadTags {
+			splitTag := strings.Split(tag, "=")
+			key, value := splitTag[0], splitTag[1]
+			hvmTagMap[key] = value
+		}
+		err = API.CreateTags([]string{hvmID}, hvmTagMap)
+
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "unable to add tags: %v\n", err)
+			os.Exit(1)
```

Comment:
This function expects error returns. Would it make more sense to return the error?

Reply:
Using `os.Exit`'s in `ore` is pretty normal (though most functions don't return `error`'s inside of `ore`). This is following the rest of the checks inside of this function.

Reply:
Returning an error from the run function causes the usage message to be displayed, which doesn't make sense for actual errors.